code/datums/keybinding/carbon.dm

```diff
@@ -20,82 +20,3 @@
 	var/mob/living/carbon/C = user.mob
 	C.toggle_throw_mode()
 	return TRUE
-
-
-/datum/keybinding/carbon/select_help_intent
-	hotkey_keys = list("1")
-	name = "select_help_intent"
-	full_name = "Select help intent"
-	description = ""
-	category = CATEGORY_CARBON
-	keybind_signal = COMSIG_KB_CARBON_SELECTHELPINTENT_DOWN
-
-/datum/keybinding/carbon/select_help_intent/down(client/user)
-	. = ..()
-	if(.)
-		return
-	user.mob?.a_intent_change(INTENT_HELP)
-	return TRUE
-
-
-/datum/keybinding/carbon/select_disarm_intent
-	hotkey_keys = list("2")
-	name = "select_disarm_intent"
-	full_name = "Select disarm intent"
-	description = ""
-	category = CATEGORY_CARBON
-	keybind_signal = COMSIG_KB_CARBON_SELECTDISARMINTENT_DOWN
-
-/datum/keybinding/carbon/select_disarm_intent/down(client/user)
-	. = ..()
-	if(.)
-		return
-	user.mob?.a_intent_change(INTENT_DISARM)
-	return TRUE
-
-
-/datum/keybinding/carbon/select_grab_intent
-	hotkey_keys = list("3")
-	name = "select_grab_intent"
-	full_name = "Select grab intent"
-	description = ""
-	category = CATEGORY_CARBON
-	keybind_signal = COMSIG_KB_CARBON_SELECTGRABINTENT_DOWN
-
-/datum/keybinding/carbon/select_grab_intent/down(client/user)
-	. = ..()
-	if(.)
-		return
-	user.mob?.a_intent_change(INTENT_GRAB)
-	return TRUE
-
-
-/datum/keybinding/carbon/select_harm_intent
-	hotkey_keys = list("4")
-	name = "select_harm_intent"
-	full_name = "Select harm intent"
-	description = ""
-	category = CATEGORY_CARBON
-	keybind_signal = COMSIG_KB_CARBON_SELECTHARMINTENT_DOWN
-
-/datum/keybinding/carbon/select_harm_intent/down(client/user)
-	. = ..()
-	if(.)
-		return
-	user.mob?.a_intent_change(INTENT_HARM)
-	return TRUE
-
-/datum/keybinding/carbon/give
-	hotkey_keys = list("G")
-	name = "Give_Item"
-	full_name = "Give item"
-	description = "Give the item you're currently holding"
-	keybind_signal = COMSIG_KB_CARBON_GIVEITEM_DOWN
-
-/datum/keybinding/carbon/give/down(client/user)
-	. = ..()
-	if(.)
-		return
-	var/mob/living/carbon/C = user.mob
-	C.give()
-	return TRUE
```

Comment:
👀

Reply:
the lord giveth, and the lord taketh away